internal/business/persisted_operations/dir_loader.go

```diff
@@ -32,20 +33,23 @@ func (d *DirLoader) Load(ctx context.Context) (map[string]string, error) {
 		}
 	}
 
-	var result map[string]string
+	var result = make(map[string]string)
```

Comment:
Any reason why you changed the declaration into an initialization?

Reply:
i needed that when i was adding items to it, through my own parsing, no longer relevant. I'll revert it